### PR TITLE
Added example for checking if two strings are a Permutation

### PR DESCRIPTION
**Pull request by harrisonbrock**

### CODE REVIEW

File: topics/go/algorithms/strings/permuation/permutation.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	// If the length are not equal they cannot be permutation.
+	if len(str1) != len(str2) {
+		return false
+	}
+
+	// Create slices for each input string.
+	s1 := strings.Split(str1, "")
+	s2 := strings.Split(str2, "")
+
+	// Sort the runes
```
Comment (ardan-bkennedy) [inline topics/go/algorithms/strings/permuation/permutation.go:20]: Fix the comment.

File: topics/go/algorithms/strings/permuation/permutation_test.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	}
+
+	for _, tt := range permutationTests {
+		got := strings.IsPermutation(tt.input, tt.input2)
+		if got != tt.expected {
+			t.Logf("\t%s\tString is a palindrome: %s\n.", failed, tt.input)
+			t.Fatalf("\t\tGot %v, Expected %v.", got, tt.expected)
+		}
+		t.Logf("\t%s\tString %s is a palindrome.", succeed, tt.input)
+
```
Comment (ardan-bkennedy) [inline topics/go/algorithms/strings/permuation/permutation_test.go:32]: Remove the extra line space.

File: topics/go/algorithms/strings/permuation/permutation.go [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package strings
+
+import (
+	"sort"
+	"strings"
+)
+
+// IsPermutation check is two strings are permutations.
```
Comment (ardan-bkennedy) [inline topics/go/algorithms/strings/permuation/permutation.go:8]: Read the function description.  I think it is. `check if two strings` not is

File: topics/go/algorithms/strings/permuation/permutation.go [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+func IsPermutation(str1, str2 string) bool {
+
+	// If the length are not equal they cannot be permutation.
+	if len(str1) != len(str2) {
+		return false
+	}
+
+	// Create slices for each input string.
+	s1 := strings.Split(str1, "")
```
Comment (ardan-bkennedy) [inline topics/go/algorithms/strings/permuation/permutation.go:17]: Describe what this does better.

Are you breaking each character out? Describe what this is doing?

File: topics/go/algorithms/strings/permuation/permutation.go (hunk 1 above)
Comment (harrisonbrock) [reply topics/go/algorithms/strings/permuation/permutation.go:20]: Fixed

File: topics/go/algorithms/strings/permuation/permutation_test.go (hunk 2 above)
Comment (harrisonbrock) [reply topics/go/algorithms/strings/permuation/permutation_test.go:32]: Fixed

File: topics/go/algorithms/strings/permuation/permutation.go (hunk 3 above)
Comment (harrisonbrock) [reply topics/go/algorithms/strings/permuation/permutation.go:8]: Fixed

File: topics/go/algorithms/strings/permuation/permutation.go (hunk 4 above)
Comment (harrisonbrock) [reply topics/go/algorithms/strings/permuation/permutation.go:17]: Fixed

File: topics/go/algorithms/strings/permuation/permutation.go (hunk 3 above)
Comment (ardan-bkennedy) [reply topics/go/algorithms/strings/permuation/permutation.go:8]: I don't see any of these changes when I view the file?

Comment (harrisonbrock) [reply topics/go/algorithms/strings/permuation/permutation.go:8]: That's funny it's not showing up. https://github.com/ardanlabs/gotraining/blob/permutation/topics/go/algorithms/strings/permuation/permutation.go

Comment (ardan-bkennedy) [reply topics/go/algorithms/strings/permuation/permutation.go:8]: check your commit/push

Comment (ardan-bkennedy) [reply topics/go/algorithms/strings/permuation/permutation.go:8]: delete the PR and re-create it.

Keep comment within 80 columns of width.

**Pull request closed without merging**